builder/digitalocean/builder.go

```diff
@@ -56,6 +56,12 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 		client.BaseURL = u
 	}
 
+	if b.config.depToken {
+		ui.Say("[WARN] The DIGITALOCEAN_API_TOKEN environment variable is deprecated " +
+			"and will produce an error in future versions of the DigitalOcean Packer plugin. " +
+			"Please use either DIGITALOCEAN_TOKEN or DIGITALOCEAN_ACCESS_TOKEN moving forward.")
+	}
```

Comment:
I think that you shouldn't need to do that, and that Packer core should display the warnings here. Was it not the case ?